src/ir/control.rs

```diff
@@ -104,6 +104,12 @@ impl InvIdx {
         inv.inst
     }
 
+    /// The times the invoke uses, along with the EventBind infos
+    pub fn times(self, ctx: &impl Ctx<Invoke>) -> Vec<(TimeIdx, InfoIdx)> {
```

Comment:
This is a little funky function to expose in the interface. Are you expecting it to be used in other places? Otherwise we can probably do:
```
comp.get(inv).events...
```

If we decide to keep it, pro-tip about interface design, instead of `.collect` in this method, you can actually return the iteration directly using:
```
times(...) -> impl Iterator<(TimeIdx, InfoIdx)> 
```

This is useful because often times, when you get a `Vec` out, you immediately want to iterate using it anyways.